sdk/contentsafety/azure-ai-contentsafety/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
 
 ### Features Added
-
+- Support AAD Authentication
+- Support 8 severity level for text:analyze API
 ### Breaking Changes
-
+- Contract change for AnalyzeText, AnalyzeImage, Blocklist management related methods
```

Comment:
Please give a detailed changelog. One line of "contract change" is not useful for customer, when SDK got lots of breaking changes.

Reply:
added more details